easybuild/easyconfigs/r/requests/requests-2.31.0-foss-2022a.eb

```diff
@@ -0,0 +1,23 @@
+easyblock = 'PythonPackage'
+
+name = 'requests'
+version = '2.31.0'
+
+homepage = 'https://pypi.python.org/pypi/requests/%(version)s'
+description = """Python http for humans"""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
```

Comment:
this is strange
in the setup.cfg file there is the requirement `requests==2.28.2`, whereas you are using `2.31.0`,
and `requests` is already included in the Python installation itself, with `2.27.1`,
what's weird is that the sanity check succeeded for me, even with the version mismatch.

Reply:
I don't really understand how that's possible either...

But, I also think we should just replace `requests==2.28.2` with `requests`, and use the `requests` provided by Python, rather than including a custom easyconfig for a newer version.

I'm 99% sure `requests` 2.27.1 works just fine for `MONAI-Label`.

Reply:
yes, i agree, the exact `requests` is probably not very important

Reply:
Is `setup.cfg` being overruled by something else, perhaps?